Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
Not sure what you mean with providing a config.py / accounts.csv. It doesn't need to be part of the image, you can easily have those files in a local folder and user docker run -v option in order to share those files between your local filesystem and docker container.